Mathlib/Data/PFunctor/Univariate/Basic.lean

```diff
@@ -39,37 +39,41 @@ namespace PFunctor
 instance : Inhabited PFunctor :=
   ⟨⟨default, default⟩⟩
 
-variable (P : PFunctor) {α β : Type u}
+variable (P : PFunctor.{u}) {α : Type v₁} {β : Type v₂}
```

Comment:
```suggestion
variable (P : PFunctor.{u}) {α β : Type*}
```

I think this should have the same effect, but is a bit more succinct

Reply:
I want to fix the order of universe variables as `u, v₁, v₂`.
With `Type*`, the order of universe variables becomes `v₁, v₂, u`.